### PR TITLE
removed localization in models

### PR DESCRIPTION
**Pull request by rhimmelbauer**

### CODE REVIEW

File: vendor/models/price.py [hunk 1]
```diff
@@ -10,12 +10,12 @@
 #########
 
 class Price(models.Model):
-    offer = models.ForeignKey("vendor.Offer", verbose_name=_("Offer"), on_delete=models.CASCADE, related_name="prices")
+    offer = models.ForeignKey("vendor.Offer", verbose_name="Offer", on_delete=models.CASCADE, related_name="prices")
     cost = models.FloatField(_("Cost"), blank=True, null=True)
     currency = models.CharField(_("Currency"), max_length=4, choices=CURRENCY_CHOICES, default=DEFAULT_CURRENCY)
     start_date = models.DateTimeField(_("Start Date"), help_text="When should the price first become available?")
     end_date = models.DateTimeField(_("End Date"), blank=True, null=True, help_text="When should the price expire?")
-    priority = models.IntegerField(_("Priority"), help_text=_("Higher number takes priority"), blank=True, null=True)
```
Comment (mgcamisa) [inline vendor/models/price.py:18]: help text should remain translated

**Review by mgcamisa — COMMENTED** [review]
If we're removing the localization in the models, we need to make sure any forms that use them have localized field names, otherwise they will not translate. Also unsure about the __str__ functions having english text only, but I'm not sure how best to handle that case; currently people in Japan will see things like their Invoice objects in English. Maybe @renderbox has some input?

File: vendor/models/invoice.py [hunk 2]
```diff
@@ -64,8 +64,8 @@ class Meta:
 
     def __str__(self):
         if not self.profile.user:
-            return _("New Invoice")
-        return _("{username} Invoice ({time})").format(username=self.profile.user.username, time=self.created.strftime('%Y-%m-%d %H:%M'))
+            return "New Invoice"
+        return "{username} Invoice ({time})".format(username=self.profile.user.username, time=self.created.strftime('%Y-%m-%d %H:%M'))
```
Comment (mgcamisa) [inline vendor/models/invoice.py:68]: example of english __str__ function

File: vendor/models/price.py (hunk 1 above)
Comment (rhimmelbauer) [reply vendor/models/price.py:18]: super localized

File: vendor/models/address.py [hunk 3]
```diff
@@ -46,7 +46,7 @@ class Address(models.Model):
         Address(): Returns an instance of the Address model
     """
     name = models.CharField(_("Address Name"), max_length=80, blank=True)                                           # If there is only a Product and this is blank, the product's name will be used, oterhwise it will default to "Bundle: <product>, <product>""
-    profile = models.ForeignKey(CustomerProfile, verbose_name=_("Customer Profile"), null=True, on_delete=models.CASCADE, related_name="addresses")
+    profile = models.ForeignKey(CustomerProfile, verbose_name="Customer Profile", null=True, on_delete=models.CASCADE, related_name="addresses")
```
Comment (renderbox) [inline vendor/models/address.py:49]: Field labels should still be localized.

File: vendor/models/base.py [hunk 4]
```diff
@@ -41,10 +41,10 @@ class ProductModelBase(CreateUpdateModelBase):
     '''
     This is the base class that all Products should inherit from.
     '''
-    sku = models.CharField(_("SKU"), max_length=40, unique=True, blank=True, null=True, help_text=_("User Defineable SKU field"))   # Needs to be autogenerated by default, and unique from the PK
+    sku = models.CharField("SKU", max_length=40, unique=True, blank=True, null=True, help_text=_("User Defineable SKU field"))   # Needs to be autogenerated by default, and unique from the PK
```
Comment (renderbox) [inline vendor/models/base.py:44]: Field labels should still be localized.

File: vendor/models/base.py [hunk 5]
```diff
@@ -41,10 +41,10 @@ class ProductModelBase(CreateUpdateModelBase):
...
     uuid = models.UUIDField(default=uuid.uuid4, editable=False, unique=True)                                           # Used to track the product
     name = models.CharField(_("Name"), max_length=80, blank=False)
-    site = models.ForeignKey(Site, verbose_name=_("Site"), on_delete=models.CASCADE, default=settings.SITE_ID, related_name="products")        # For multi-site support
+    site = models.ForeignKey(Site, verbose_name="Site", on_delete=models.CASCADE, default=settings.SITE_ID, related_name="products")        # For multi-site support
```
Comment (renderbox) [inline vendor/models/base.py:47]: Field labels should still be localized.

File: vendor/models/invoice.py [hunk 6]
```diff
@@ -32,9 +32,9 @@ class InvoiceStatus(models.IntegerChoices):
         COMPLETE = 50, _("Complete")    # Payment Processor Completed Transaction.
         REFUNDED = 60, _("Refunded")    # Invoice Refunded to client. 
 
-    uuid = models.UUIDField(_("UUID"), default=uuid.uuid4, editable=False, unique=True)
-    profile = models.ForeignKey("vendor.CustomerProfile", verbose_name=_("Customer Profile"), null=True, on_delete=models.CASCADE, related_name="invoices")
-    site = models.ForeignKey(Site, verbose_name=_("Site"), on_delete=models.CASCADE, default=set_default_site_id, related_name="invoices")                      # For multi-site support
+    uuid = models.UUIDField("UUID", default=uuid.uuid4, editable=False, unique=True)
```
Comment (renderbox) [inline vendor/models/invoice.py:35]: Field labels should still be localized.

File: vendor/models/invoice.py [hunk 7]
```diff
@@ -124,13 +124,13 @@ class OrderItem(CreateUpdateModelBase):
...
-    invoice = models.ForeignKey("vendor.Invoice", verbose_name=_("Invoice"), on_delete=models.CASCADE, related_name="order_items")
-    offer = models.ForeignKey("vendor.Offer", verbose_name=_("Offer"), on_delete=models.CASCADE, related_name="order_items")
+    invoice = models.ForeignKey("vendor.Invoice", verbose_name="Invoice", on_delete=models.CASCADE, related_name="order_items")
+    offer = models.ForeignKey("vendor.Offer", verbose_name="Offer", on_delete=models.CASCADE, related_name="order_items")
     quantity = models.IntegerField(_("Quantity"), default=1)
 
     class Meta:
-        verbose_name = _("Order Item")
-        verbose_name_plural = _("Order Items")
+        verbose_name = "Order Item"
```
Comment (renderbox) [inline vendor/models/invoice.py:135]: These are correct to have the localization removed in the Meta.

File: vendor/models/invoice.py [hunk 8]
```diff
@@ -65,7 +65,7 @@ class Meta:
     def __str__(self):
         if not self.profile.user:
             return _("New Invoice")
-        return _("{username} Invoice ({time})").format(username=self.profile.user.username, time=self.created.strftime('%Y-%m-%d %H:%M'))
+        return str(self.profile.user.username) + " Invoice (" + self.created.strftime('%Y-%m-%d %H:%M') + ")"
```
Comment (renderbox) [inline vendor/models/invoice.py:68]: No localization inside of the \_\_str\_\_()

**Review by renderbox — CHANGES_REQUESTED** [review]
Make sure to restore the localization in the fields but remove them from the Meta and the \_\_str\_\_() calls.

File: vendor/models/address.py (hunk 3 above)
Comment (rhimmelbauer) [reply vendor/models/address.py:49]: added localization

File: vendor/models/base.py (hunk 4 above)
Comment (rhimmelbauer) [reply vendor/models/base.py:44]: added localization

File: vendor/models/base.py (hunk 5 above)
Comment (rhimmelbauer) [reply vendor/models/base.py]: added localization

File: vendor/models/invoice.py (hunk 6 above)
Comment (rhimmelbauer) [reply vendor/models/invoice.py]: added localization

File: vendor/models/invoice.py (hunk 8 above)
Comment (rhimmelbauer) [reply vendor/models/invoice.py:68]: added get_invoice_display method

File: vendor/models/address.py (hunk 3 above)
Comment (mgcamisa) [reply vendor/models/address.py:49]: looks like this one is still missing

**Review by mgcamisa — APPROVED** [review]

**Pull request merged**